### PR TITLE
[zh-cn] Update DOMException index

### PR DESCRIPTION
**Pull request by jasonren0403**
### Description

fix unfinished translations

### CODE REVIEW

File: files/zh-cn/web/api/domexception/index.md [hunk 1]
```diff
@@ -5,94 +5,100 @@ slug: Web/API/DOMException
 
 {{ APIRef("DOM") }}
 
-**`DOMException`** 接口代表调用方法或访问 Web API 属性时发生的异常事件（被称为**异常**，**exception**）。这基本上是在 Web API 中如何描述错误情况的
+**`DOMException`** 接口代表调用方法或访问 Web API 属性时发生的不正常事件（被称为**异常**，**exception**）。这就是 Web API 中描述错误条件的方式。
 
-每个异常都有一个名称 **name**，一个采用骆驼式命名法的简短字符串，用于描述识别错误或异常情况。
+每个异常都有一个名称 **name**，一个采用驼峰命名法的简短字符串，用于描述识别错误或异常情况。
```
Comment (yin1999) [inline files/zh-cn/web/api/domexception/index.md:10]:
```suggestion
每个异常都有一个**名称**（name），一个采用驼峰命名法的简短字符串，用于描述识别错误或异常情况。
```

File: files/zh-cn/web/api/domexception/index.md [hunk 2]
```diff
@@ -5,94 +5,100 @@ slug: Web/API/DOMException
...
-- {{domxref("DOMException.DOMException()", "DOMException()")}} {{experimental_inline}}
+- {{domxref("DOMException.DOMException()", "DOMException()")}}
   - : 返回一个包含指定消息和名称的 `DOMException` 对象。
 
-## 属性
+## 实例属性
 
 - {{domxref("DOMException.code")}} {{deprecated_inline}} {{readOnlyInline}}
-  - : 返回一个 `short`，包含错误代码常量中的一个，或者返回 `0`，如果没有匹配的话。这个字段由于历史原因被使用。现在不再使用这个新的 DOM 异常：他们把这个信息放入 {{domxref("DOMException.name")}} 属性。
+  - : 返回任意一个历史错误代码常量，如果不匹配则返回 `0`。
```
Comment (yin1999) [inline files/zh-cn/web/api/domexception/index.md:22]: 遗留错误代码常量

Comment (jasonren0403) [inline files/zh-cn/web/api/domexception/index.md:22]:
```suggestion
  - : 返回任意一个遗留错误代码常量，如果不匹配则返回 `0`。
```

**Review by yin1999 — APPROVED** [review]
Thanks @jasonren0403

**Pull request merged**